docs/cli/lokoctl.md

```diff
@@ -21,6 +21,7 @@ Manage Lokomotive clusters
 ### SEE ALSO
 
 * [lokoctl cluster](lokoctl_cluster.md)	 - Manage a cluster
+* [lokoctl completion](lokoctl_completion.md)	 - Generate autocompletion script for the specified shell
```

Comment:
I'd actually put this all in a single commit, so if one runs the build per commit, it stays atomic.